djangocms_navigation/admin.py

```diff
@@ -19,9 +22,84 @@ def save_model(self, request, obj, form, change):
             )
         super().save_model(request, obj, form, change)
 
+    def get_list_display(self, request):
+        list_display = ["title", "get_menuitem_link"]
+        return list_display
+
+    def get_menuitem_link(self, obj):
+        object_preview_url = reverse(
+            "admin:{app}_menuitem_list".format(
```

Comment:
```suggestion
             "admin:{app}_{model}_list".format(
```

Reply:
the model here is 'menucontent` hence I hardcorded to `menuitem`